packages/hint/src/lib/cli/analyze.ts

```diff
@@ -361,6 +361,7 @@ export default async (actions: CLIOptions): Promise<boolean> => {
         const formatterOptions: FormatterOptions = {
             config: userConfig || undefined,
             date,
+            output: actions.output,
```

Comment:
How about resolving this to an absolute path against the current working directory here so we don't have to do it in each formatter?

Reply:
done!